### ISSUE TITLE
ios7  crash

### ISSUE THREAD

**Issue by Qiunee** [sakkaras/SKSTableView#36]
When i use in IOS7.

**Comment by haostrive** [comment]
- (NSInteger)subRow
{
    id indexpath = [NSIndexPath class];
    id subRowObj = objc_getAssociatedObject(indexpath, SubRowObjectKey);
    return [subRowObj integerValue];
}

- (void)setSubRow:(NSInteger)subRow
{
    id subRowObj = [NSNumber numberWithInteger:subRow];
    id indexpath = [NSIndexPath class];
    objc_setAssociatedObject(indexpath, SubRowObjectKey, subRowObj, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
}
My solution but i don't know whether it is the best

**Comment by mariovillamizar** [comment]
Yes, that's working for me. Please update the control with this change.
